marked-element.html

```diff
@@ -118,6 +118,19 @@
         observer: 'render',
         type: Boolean,
         value: false
+      },
+      /**
+       * Callback function invoked by Marked after HTML has been rendered.
+       * It must take two arguments: err and text and must return the resulting text.
+       */
+      callback: {
+        observer: 'render',
+        type: Function,
+        value: function() {
+          return function(err, text) {
+            return text;
```

Comment:
Yes I have turned it into null per https://github.com/chjj/marked/blob/master/lib/marked.js#L1148. Sorry for the long wait, I was on vacation :joy: